_docs/reference/istioctl.md

```diff
@@ -9,22 +9,22 @@ layout: docs
 type: markdown
 ---
 
-`istioctl` is a command line interface for managing an Istio service mesh.  This overview covers
+**istioctl** is a command line interface for managing an Istio service mesh.  This overview covers
 syntax, describes command operations, and provides examples.
 
 # Syntax
 
-Istioctl commands follow the syntax:
+*istioctl* commands follow the syntax:
 
-```
+```shell
 istioctl <command> [targets] [flags]
 ```
 
-where `command`, `targets` and `flags` are:
+where *command*, *targets* and *flags* are:
 
 * **command**: the operation to perform, such as `create`, `delete`, `replace`, or `list`.
```

Comment:
shouldn't this be create, get, replace, or delete. (essentially change list to get, and reorder)